src/store/types.d.ts

```diff
@@ -0,0 +1,41 @@
+import { Action as ReduxAction, compose } from 'redux';
+
+export type TAppState = {
+  user: TUserState;
+  game: TGameState;
+};
+
+export type TUserData = {
+  id: number;
+  login: string;
+  avatar: string | null;
+  first_name: string;
+  second_name: string;
+  display_name: string | null;
+  email: string;
+  phone: string;
+};
+
+export type TUserState = {
+  isAuth: boolean;
+  pending: boolean;
+  error?: Error;
```

Comment:
❗️
В одном из комментов найдёшь аргументацию почему лучше так)
```suggestion
  error: Error | null;
```